core/main_test.go

```diff
@@ -15,6 +15,7 @@ func TestMain(m *testing.M) {
 	opts := []goleak.Option{
 		// No good way to shut down these goroutines:
 		goleak.IgnoreTopFunction("github.com/ava-labs/coreth/core/state/snapshot.(*diskLayer).generate"),
+		goleak.IgnoreTopFunction("github.com/ava-labs/libevm/core.(*txSenderCacher).cache"),
```

Comment:
importing this actually starts a bunch of goroutines. kind of silly but not sure what we can do about it.

Reply:
Yeah this isn't something we need to manage here. Maybe in `libevm` if we think it's an issue, but ideally in `geth` (as a low priority).